reliability-engineering/terraform/modules/concourse-worker-pool/autoscaling-group.tf

```diff
@@ -1,18 +1,18 @@
 resource "aws_autoscaling_group" "concourse_worker" {
   name                = "${var.deployment}-${var.name}-concourse-worker"
-  max_size            = "${var.desired_capacity * 2}"
+  max_size            = var.desired_capacity * 2
   min_size            = 0
-  desired_capacity    = "${var.desired_capacity}"
-  vpc_zone_identifier = ["${var.subnet_ids}"]
+  desired_capacity    = var.desired_capacity
+  vpc_zone_identifier = var.subnet_ids
 
-  launch_template = {
-    id      = "${aws_launch_template.concourse_worker.id}"
-    version = "$$Latest"
```

Comment:
same here

Reply:
looks like it: https://www.terraform.io/docs/providers/aws/r/autoscaling_group.html#with-latest-version-of-launch-template